x-pack/auditbeat/module/system/packages/rpm_others.go

```diff
@@ -0,0 +1,11 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build !linux
+
+package packages
+
+func listRPMPackages() ([]*Package, error) {
+	return rpmPackagesByExec()
```

Comment:
Would it make sense to just stub this out rather than providing an implementation?